lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -239,11 +239,28 @@ describe('BasicLtiLaunchApp', () => {
     });
   });
 
-  // FIXME-A11Y
-  it.skip(
+  it(
     'should pass a11y checks',
-    checkAccessibility({
-      content: () => renderLtiLaunchApp(),
-    })
+    checkAccessibility([
+      {
+        content: () => renderLtiLaunchApp(),
+      },
+      {
+        name: 'LMS grader mode',
+        content: () => {
+          // Override the fake config for this one test.
```

Comment:
Note: I checked checkAccessibility, looks like order honored, which if for any reason changes, then this test would not reflect its intent.

We may want to implement a beforeEach/afterEach sorta thing with an array of scenarios?

Reply:
> Note: I checked checkAccessibility, looks like order honored, which if for any reason changes, then the previous test may not reflect its intent.

Sorry, not following—could you rephrase?

Reply:
@lyzadanger 
So, because these act like more than one test (array of tests) they can and often need to have their own config. But there is no `beforeEach` equivalent to reset all the params before each test. So when we override a fake object like `fakeConfig`, then that won't change back before the next test. In the is case, it happens to be the last test so its okay, but that is not obvious! I think we'll run into this a little bit with the axe tests and we may need some thinking about how to solve it elegantly.